packages/app-desktop/gui/Sidebar/NotebookAndTagList.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { AppState } from '../../app.reducer';
+import ItemList from '../ItemList';
+import { connect } from 'react-redux';
+import useSidebarListData from './hooks/useSidebarListData';
+import { FolderEntity, TagsWithNoteCountEntity } from '@joplin/lib/services/database/types';
+import useOnRenderItem from './hooks/useOnRenderItem';
+import { Dispatch } from 'redux';
+import { PluginStates } from '@joplin/lib/services/plugins/reducer';
+import { useMemo, useRef, useState } from 'react';
+import useElementHeight from '../hooks/useElementHeight';
+import useSelectedSidebarIndex from './hooks/useSelectedSidebarIndex';
+import useOnSidebarKeyDownHandler from './hooks/useOnSidebarKeyDownHandler';
+import { ListItem } from './types';
+import useFocusHandler from './hooks/useFocusHandler';
+
+interface Props {
+	dispatch: Dispatch;
+	themeId: number;
+	tags: TagsWithNoteCountEntity[];
+	folders: FolderEntity[];
+	notesParentType: string;
+	selectedTagId: string;
+	selectedFolderId: string;
+	selectedSmartFilterId: string;
+	collapsedFolderIds: string[];
+	folderHeaderIsExpanded: boolean;
+	tagHeaderIsExpanded: boolean;
+	plugins: PluginStates;
+}
+
+
+const NotebookAndTagList: React.FC<Props> = props => {
```

Comment:
I've made this change in most places.

For now, I've left [sidebar.spec.ts](https://github.com/laurent22/joplin/pull/10331/files#diff-5487459e210f5ab68fd5db2b378aa54b35ee4f5eb2a20014a1aee8e8fd2831dc) as-is, because Playwright is accessing user-facing labels (which use "notebook") to run the test. I can change the internal variable names there well if desired.

Reply:
I would prefer we use "folder" there too if that's ok, including in test descriptions, and even when creating folders for test (So "Folder A" instead of "Notebook A").

Basically the only time we should use the term "notebook" is in strings to be translated or in documentation. Ideally we'd change everything to use only one term or the other, but that would be more trouble than it's worth.

Reply:
Got it! This should be addressed by 97ec1a9db829b7c93e0a7375a4d79e89d224fc10.